spec/models/membership_application_spec.rb

```diff
@@ -121,6 +121,17 @@
 
   end
 
+  describe "#destroy" do
+
+    let(:application_owner) {create(:user, email: 'user_1@random.com')}
+    let(:membership_application) {create(:membership_application, user: application_owner, uploaded_files: [create(:uploaded_file, actual_file: (File.new(File.join(FIXTURE_DIR, 'image.jpg'))))])}
+
+    it 'destroys a membershipapplication' do
+      expect {membership_application.destroy}.not_to raise_exception
+    end
+
+  end
```

Comment:
Need tests to prove that the uploaded files were actually deleted.
(Just testing that an exception isn't raised isn't evidence that things work as expected.)